public/button.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+$(function() {
+
```

Comment:
I could argue that you still use a little too many line breaks for white space in your code. Consolidate some more!